Sources/DataPipeline/Config/SDKConfigBuilder.swift

```diff
@@ -137,12 +168,22 @@ public class SDKConfigBuilder {
         return self
     }
 
+    @available(iOSApplicationExtension, unavailable)
     public func build() -> SDKConfigBuilderResult {
-        // create `SdkConfig`` from given configurations
+        // create `SdkConfig` from given configurations
         let sdkConfig = SdkConfig.Factory.create(
             logLevel: logLevel
         )
 
+        // create plugins based on given configurations
+        var configuredPlugins: [Plugin] = []
```

Comment:
maybe on `de init` we should empty the array?

Reply:
Might not be needed here as it is in function. And can't add to `DataPipelineConfigOptions` as it is struct.